### DISCUSSION TITLE
Meian SW02 water leak detector works out-of-the-box :)

### DISCUSSION THREAD

**Discussion by Westcott1** [Koenkk/zigbee2mqtt#16124] [Devices]
[https://www.aliexpress.com/item/1005004707117249.html?spm=a2g0o.order_list.order_list_main.5.d43918026PLcLV](url)

Zigbee Model TS0207
Zigbee Manufacturer
 _TZ3000_kyb656no
Description
 Water leak detector
Support status
 Supported

Manufacturer
 TuYa
Model
 TS0207_water_leak_detector

**Comment by jeroenvermeulen** [comment]
Took me a while to find:
There is a button hidden under the top cover. 
You just need to press on the water drop icon on the top to join the water leak sensor to your Zigbee network.
